packages/shared_preferences/lib/shared_preferences_tizen.dart

```diff
@@ -26,51 +26,49 @@ class SharedPreferencesPlugin extends SharedPreferencesStorePlatform {
     final String key = pKey.toDartString();
     int ret;
 
-    final Pointer<Int8> pBool = malloc();
-    ret = bindings.getBoolean(pKey, pBool);
-    final bool boolValue = pBool.value == 1;
-    malloc.free(pBool);
-    if (ret == 0) {
-      _cachedPreferences![key] = boolValue;
-      return 1;
-    }
+    using((Arena arena) {
+      // Try as boolean type
```

Comment:
Please either remove this or add `.` at the end.

Reply:
my mistake.. :(